answer/naka1998/JS/10/3sec.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <style>
+        button{
+            color: white;
+            font-size: 50px;
+            background-color: red;
+            border-radius: 20px;
+        }
+    </style>
+</head>
+<body>
+    <button onclick="alert_button()">非常ボタン</button>
+    <script>
+        function alert_button(){
```

Comment:
```suggestion
        function alertButton(){
```

javascriptでは変数名。関数名はcamelCaseにする習慣があります。
refs: http://cou929.nu/data/google_javascript_style_guide/#id28

習わしにしたがって↑のようにしましょう